### PR TITLE
ci: [FX-4126] write temploy link to the PR description

### PR DESCRIPTION
**Pull request by TomasSlama**
[FX-4126]

### Description

We need to show link to temploy

### How to test

- [Temploy](https://picasso.toptal.net/FX-NULL-temploy-link)
- FIXME: Add the steps describing how to verify your changes

### Development checks

- [x] Ensure that deployed demo has expected results and good examples
- [x] Self reviewed

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-4126]: https://toptal-core.atlassian.net/browse/FX-4126?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -148,5 +148,9 @@
     "react-dom": "^18.2.0",
     "react-test-renderer": "^18.2.0",
     "**/quill-emoji/**/clean-css": "^4"
+  },
+  "dependencies": {
```
Comment (sashuk) [inline package.json:152]: NIT: maybe to move it to `devDependencies`?

**Comment by sashuk** [comment]
According to the history of changes, the `- Temploy` was updated, but somehow the Jira link was reverted

Screenshot 2023-07-04 at 20 27 38

**Comment by dmaklygin** [comment]
> According to the history of changes, the `- Temploy` was updated, but somehow the Jira link was reverted
> 
> Screenshot 2023-07-04 at 20 27 38

I think, it because I edited the PR during GH Workflow execution. It had a previous version

**Review by sashuk — APPROVED** [review]

**Comment by sashuk** [comment]
@toptal-anvil ping reviewers

**Comment by sashuk** [comment]
@dmaklygin I've tried to ping reviewers in https://github.com/toptal/picasso/pull/3621#issuecomment-1622171986 and it failed due to nobody's review needed according to the (Anvil uses it for getting list of reviewers)

Screenshot 2023-07-05 at 19 17 47

However, pull requests have a requirement to have at least 2 reviews, so every time we need to go and manually add the FX Team as a reviewer

Screenshot 2023-07-05 at 19 18 47

Do you maybe know if this has been already addressed or there is a ticket for this?

**Comment by dmaklygin** [comment]
@toptal-anvil ping reviewers

**Review by mkrl — APPROVED** [review]

**Pull request merged**